omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -1186,7 +1186,7 @@
 
                 var formattedDate = changeFieldDateToJavascriptDate(selectedDate);
 
-                <includeIf velocityTest="$fn.globalProperty('ugandaemr.maximumPatientsPerDay') != ''">
+                <includeIf velocityTest="$fn.globalProperty('ugandaemr.enableMaximumPatientsPerDay').trim() == true">
```

Comment:
@slubwama This check has to be against the text true, does not work when the appointment checks are enabled